blueflood-core/src/main/java/com/rackspacecloud/blueflood/rollup/SlotKey.java

```diff
@@ -135,6 +136,33 @@ public static SlotKey parse(String string) {
     }
 
     /**
+     * Same as the method {@link #getChildrenKeys()} except this method returns only
+     * the children corresponding to the destination granularity.
+     *
+     * For example, a slot key of granularity
+     * {@link Granularity#MIN_1440 MIN_1440}, for destination granularity of
+     * {@link Granularity#MIN_60 MIN_60}, will have 6*4=24 children.
```

Comment:
There is getChildrenKeys above which would list children of all sublevels. This method uses that and only returns the level we are interested in. I just followed the terminology used there.